nuitka/plugins/standard/PkgResourcesPlugin.py

```diff
@@ -77,6 +77,20 @@ def onModuleSourceCode(self, module_name, source_code):
 
                 source_code = source_code.replace(match[0], value)
 
+            for match in re.findall(
+                r"""\b(pkg_resources\.require\(\s*['"](.*?)['"]\s*\))""",
+                source_code,
+            ):
+                # Explicitly call the require function at Nuitka compile, and
+                # if it fails remove it so that it doesn't fail at execution
+                try:
+                    self.pkg_resources.require(match[1])
+                except self.pkg_resources.ResolutionError:
+                    raise self.pkg_resources.ResolutionError(
```

Comment:
I wouild do a `self.sysexit()` style program termination here, but that's definitely OK.

Reply:
I will add this is a cleanup commit myself.